precompiles/outposts/stride/IStrideOutpost.sol

```diff
@@ -11,6 +11,19 @@ IStrideOutpost constant STRIDE_OUTPOST_CONTRACT = IStrideOutpost(
     STRIDE_OUTPOST_ADDRESS
 );
 
+/// @dev AutopilotParams is a struct containing the parameters for a liquid stake and redeem transactions.
+/// @param receiver - The address on the Evmos chain that will receive LSD.
```

Comment:
if this struct is used also for the redeem, then address will not receive an LSD in this case.